vllm/model_executor/layers/logits_processor.py

```diff
@@ -70,6 +70,11 @@ def _get_logits(self, hidden_states: torch.Tensor, embedding: torch.Tensor,
             logits = logits[:, :self.org_vocab_size]
         return logits
 
+    def extra_repr(self) -> str:
+        s = "vocab_size={vocab_size}, org_vocab_size={org_vocab_size}"
+        s += ", scale={scale}, logits_as_input={logits_as_input}"
+        return s.format(**self.__dict__)
```

Comment:
why not use an f-string here?

Reply:
I referred to the code from  https://github.com/pytorch/pytorch/blob/main/torch/nn/modules/sparse.py#L167. Indeed, I should ensure consistent use of f-strings :smile: , I will process this as soon as possible